assets/test/integration/helpers.js

```diff
@@ -78,3 +80,15 @@ exports.goToPage = function(uri) {
     waitForUri(uri);
   });
 };
+
+exports.presenceOf = function(obj, msg) {
+  msg = msg || '';
+  return protractor.ExpectedConditions.presenceOf(obj);
```

Comment:
I think it is supposed to be:
```javascript

exports.presenceOf = function(obj) {
  return protractor.ExpectedConditions.presenceOf(obj);
};

// example:
//  h.waitFor(h.presenceOf(page.header()))
exports.waitFor = function(item, timeout, msg) {
  timeout = timeout || 5000;
  msg = msg || '';
  return browser.wait(item, timeout, msg);
};
```
But for the most part I put the tests back to the old way.  I'll update this quick even though its not being used (until future tests).